src/DataProccesing.java

```diff
@@ -0,0 +1,95 @@
+import java.io.*;
+
+public class DataProccesing {
+
+    /**
+     * @param login    - written by user;
+     * @param password - written by user;
+     */
+    public static void dataSaving(String login, String password) {
+        File file = new File("data_users.csv");
+        try (FileWriter write = new FileWriter(file, true);
```

Comment:
You can a lot of duplicate logic in 2 classes - all related to the file processing. Can we create another object that will handle all this functionality? For example you can use name `FileRepository` 
Keep in mind that `FileRepository`  class only works with files, it doesn't know anything about waching machines or users